### DISCUSSION TITLE
[FR] Pinned Entries / quicklinks widget for dashboard

### DISCUSSION THREAD

**Discussion by proimage** [craftcms/cms#8823] [Ideas]
I'd find it very useful to have a manually-curated list of links to entries in the CP dashboard. For example, during dev phases, I would pin to the list the various test entries I've created in the various site sections. A client might want to pin an oft-updated homepage.

Basically, a Recent Entries widget, but with manual selection of entries instead of automatically by date.

**Comment by brandonkelly** [comment]
Good idea! In the meantime, check out the [DIY Widget](https://plugins.craftcms.com/diy-widget) plugin – pretty useful.
